coreax/approximation.py

```diff
@@ -180,7 +183,7 @@ def gramian_row_mean(self, x: ArrayLike, **kwargs) -> Array:
         :return: Approximation of the base kernel's Gramian row-mean
         """
         del kwargs
```

Comment:
Just as a sense check, do you know when this was introduced, and what it needs doing at the top of all of these methods? (deleting the `kwards`)?

Reply:
The kwargs parameter is for Liskov, the del is to supress pylint's complaint of an unused variable.